src/lib/toastr/toastr-service.ts

```diff
@@ -49,29 +49,31 @@ export class ToastrService {
     private _injector: Injector,
     private sanitizer: DomSanitizer,
   ) {
-    this.toastrConfig = this.applyConfig(toastrConfig);
+    const defaultConfig = new DefaultGlobalConfig;
+    this.toastrConfig = { ...defaultConfig, ...toastrConfig };
+    this.toastrConfig.iconClasses = { ...defaultConfig.iconClasses, ...toastrConfig.iconClasses };
```

Comment:
Have to manually override iconClasses to properly override the fields individually. Is there a better way of doing this?

Reply:
Nope... If we have to bump major version again in the future I'd consider making the icon classes flat and just making the names longer. I only did the config this way because the original toastr did.
```
{
timeOut: 100,
warningIconClass: 'warning',
}
```